test.init.config

```diff
@@ -1,4 +1,5 @@
 {
+    admin localhost:2999
```

Comment:
Updating the caddy dependency to `v2.6.4`, tests were failing with:
```
/coraza-caddy/caddytest.go:223: POSTed configuration isn't active
/coraza-caddy/caddytest.go:103: failed ensuring config is running: EnsureConfigRunning: POSTed configuration isn't active
```
Adding this line seems to fix the problem. The related issue should be https://github.com/caddyserver/caddy/pull/5079

Reply:
Ping @mholt

Reply:
We are going to be reworking all our tests for the 2.8 tree. (FYI)